README.md

```diff
@@ -80,33 +76,30 @@ services:
   my_another_rails:
     ...
 ```
-More `aws-rails-provisioner` examples see `./examples` (see `tiny.yml` for a minimal `aws-rails-provisioner.yml` configuration example),
-full configuration options documentation see [here](https://docs.aws.amazon.com/awsrailsprovisioner/api/).
+
+See `./examples` for more `aws-rails-provisioner` examples (see `tiny.yml` for a minimal `aws-rails-provisioner.yml` configuration example). The full configuration options are documented [here](https://docs.aws.amazon.com/awsrailsprovisioner/api/).
 
 ### Build and Deploy
```

Comment:
I may be missing some context here, but I feel like the documentation is missing information on when to run these commands (ie, do you only need to run build once?  What about if I make changes to my `aws-rails-provisioner.yml` file - do I need to run `deploy` again (or is that handled by the CICD build steps?)

Reply:
You run build to generate code then you run deploy to run the generated code. If you change the file then you need to rebuild before you redeploy. The CICD is for changing rails code only.